src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig

```diff
@@ -146,6 +146,9 @@
             <tr>
                 <td class="font-normal">
                     <span class="label status-{{ symfony_status_class[collector.symfonystate] }}">{{ symfony_status[collector.symfonystate]|upper }}</span>
+                    {% if collector.symfonylts %}
```

Comment:
Shouldn't it match the `ConfigDataCollector` key?

```suggestion
                    {% if collector.symfony_lts %}
```

Reply:
actually for me it should match the public function `isSymfonyLts` or `symfonyLts` at lease
but I saw others writen in lowercase, so I reproduce the same code :)

Reply:
Ah, it's calling the method and not the `$data` array from the `Collector`. 🤦‍♂ 

Yeah, I agree, that it really should be `symfonyLts` to match the casing of the method. 👍 (maybe we can change `symfonyEom` as well?)

Reply:
@apfelbox was not my original code, and on my old PR i was asked to rollback changes like that
so i prefered to go the same way as now :)